main.js

```diff
@@ -1,14 +1,232 @@
+  },
+  {
+    name: 'Shilla Stay',
+    city: 'Seoul',
+    country: 'South Korea',
+  },
+  {
+    name: 'San Firenze Suites',
+    city: 'Florence',
+    country: 'Italy',
+  },
+  {
+    name: 'The Andaman Resort',
+    city: 'Port Dickson',
+    country: 'Malaysia',
+  },
+  {
+    name: 'Black Penny Villas',
+    city: 'BTDC, Nuca Dua',
+    country: 'Indonesia',
+  },
+  {
+    name: 'Koko Hotel',
+    city: 'Tokyo',
+    country: 'Japan',
+  },
+  {
+    name: 'Ramada Plaza',
+    city: 'Istanbul',
+    country: 'Turkey',
+  },
+  {
+    name: 'Waikiki Resort Hotel',
+    city: 'Hawaii',
+    country: 'USA',
+  },
+  {
+    name: 'Puro Hotel',
+    city: 'Krakow',
+    country: 'Poland',
+  },
+  {
+    name: 'Asma Suites',
+    city: 'Santorini',
+    country: 'Greece',
+  },
+  {
+    name: 'Cityden Apartments',
+    city: 'Amsterdam',
+    country: 'Netherlands',
+  },
+  {
+    name: 'Mandarin Oriental',
+    city: 'Miami',
+    country: 'USA',
+  },
+  {
+    name: 'Concept Terrace Hotel',
+    city: 'Rome',
+    country: 'Italy',
+  },
+  {
+    name: 'Ponta Mar Hotel',
+    city: 'Fortaleza',
+    country: 'Brazil',
+  },
+  {
+    name: 'Four Seasons Hotel',
+    city: 'Sydney',
+    country: 'Australia',
+  },
+  {
+    name: 'Le Meridien',
+    city: 'Nice',
+    country: 'France',
+  },
+  {
+    name: 'Apart Neptun',
+    city: 'Gdansk',
+    country: 'Poland',
+  },
+  {
+    name: 'Lux Isla',
+    city: 'Ibiza',
+    country: 'Spain',
+  },
+  {
+    name: 'Nox Hostel',
+    city: 'London',
+    country: 'UK',
+  },
+  {
+    name: 'Leonardo Vienna',
+    city: 'Vienna',
+    country: 'Austria',
+  },
+  {
+    name: 'Adagio Aparthotel',
+    city: 'Edinburgh',
+    country: 'UK',
+  },
+  {
+    name: 'Steigenberger Hotel',
+    city: 'Hamburg',
+    country: 'Germany',
+  },
+];
+const find = (value) => {
+  return hotels.filter(
+    (el) => el.name === value || el.city === value || el.country === value,
+  );
+};
+
+const findString = find('Poland');
+
+console.log(
+  findString.map(({ name, city, country }) => String([name, city, country])),
+);
+
+console.log('-----');
+
+const cityByCountry = {};
+for (const { city, country } of hotels) {
+  if (!Object.hasOwn(cityByCountry, country)) {
+    cityByCountry[country] = [];
+  }
+  cityByCountry[country].push(city);
 }
+console.log(cityByCountry);
+
+console.log('------');
+
+const daysInMonth = 30;
+const daysInWeek = 7;
+const dayOfWeek = 2;
+
+const getCalendarMonth = (daysInMonth, daysInWeek, dayOfWeek) => {
+  let day = daysInMonth - dayOfWeek + 1;
+  let dates = [];
+  for (let i = 0; i < 5; i++) {
+    const week = [];
+    for (let j = 0; j < daysInWeek; j++) {
+      if (dayOfWeek > 7) {
+        return 'Error';
+      } else if (day > 30) {
+        day = 1;
+      }
+      week.push(day);
+      day++;
+    }
+    dates.push(week);
+  }
+  return dates;
+};
+
+const calendarMonth = getCalendarMonth(daysInMonth, daysInWeek, dayOfWeek);
```

Comment:
Test your functions.
Returns of getCalendarMonth(10, 2, 1) and getCalendarMonth(10, 2, 3) don't assign the task.